MAINTAINERS.md

```diff
@@ -0,0 +1,59 @@
+# Maintainers
+
+This document contains list of present maintainers, their role and process to add new maintainers.
+
+## Current Maintainers
```

Comment:
I'm unsure if this section is a good idea, though perhaps not for a good reason.  In general, people (e.g. journalists and non-contributors to the project) in my experience overestimate the role of maintainers (i.e. as power/control/gatekeepers rather than implementing the rough consensus of reviewers), and conversely, they undervalue the key vital role of the contributors who are doing consistent review and testing that shape and determine rough consensus.

Also, these roles may be fluid, require updating here, and "General" before each seems redundant.

Reply:
- Removed 'Focus Area' column 
- I understand your point about journalists however they would anyway find these names from other links which maybe misleading, outdated etc. Role of maintainers and contributors is defined in next section with 'rough consensus' part mentioned. A `maintainers.md` file without names of maintainers would be incomplete IMO